packages/@okta/vuepress-site/docs/guides/sharing-cert/overview/index.md

```diff
@@ -2,25 +2,20 @@
 title: Overview
 ---
 
-[Application key credential objects](/docs/reference/api/apps/#application-key-credential-model)
-contain an opaque key ID (`kid`) and corresponding certificate.
-To use the [API](/docs/reference/api/apps/#clone-application-key-credential)
-to share application key credentials between apps, you need to create and use a new credential in one app,
-then share and update the credential in another app.
+[Application Key Credential objects](/docs/reference/api/apps/#application-key-credential-model)
```

Comment:
I believe the link should be /docs/reference/api/apps/#application-key-credential-object (object not model at the end of the link). Also don't need a carriage return after the link as this is all one paragraph.

Reply:
It has a redirect, but I'll update it.
And there isn't a carriage return after it, trust me.